pulp_rpm/tests/functional/api/test_pulp_to_pulp.py

```diff
@@ -88,7 +88,7 @@ def do_test(self, policy):
         # Create a publication.
         publish_data = RpmRpmPublication(
             repository=repo.pulp_href,
-            metadata_checksum_type="sha1",
+            metadata_checksum_type="sha256",
```

Comment:
Actually, let's make this a "weird" checksum type like sha384

Reply:
Just so that it's definitely not the default.

Reply:
A fine idea - updated!